### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by SotoArmando**
The calculator must have a root component called App which will contain a Display component responsible for displaying the outcome of the calculations, and a ButtonPanel component which will just be a container for all the buttons. Finally each button in ButtonPanel will implement the Button component which represents a single button.

### CODE REVIEW

File: src/Components/display.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const Display = props => {
+  const { result } = props;
+  return (
+    <div className="display">
+      <span>{result}</span>
+    </div>
+  );
+};
+Display.propTypes = {
+  result: PropTypes.number.isRequired,
```
Comment (Oghenebrume50) [inline src/Components/display.js]: this should be a string see this

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED 

Hi @SotoArmando good job on this start, please make changes as outlined below and request for another review :+1: 

**RAPHAEL NORIODE** :sunglasses:

File: src/Components/display.js (hunk 1 above)
Comment (SotoArmando) [reply src/Components/display.js]: ahh thanks :)

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
-          Learn React
-        </a>
-      </header>
+      <link href="https://fonts.googleapis.com/css?family=Spartan:400,500,700&display=swap" rel="stylesheet" />
+      <Display result={0} />
+      <Buttonpanel names={['AC', '+/-', '%', '÷']} />
+      <Buttonpanel names={['7', '8', '9', 'X']} />
+      <Buttonpanel names={['4', '5', '6', '-']} />
+      <Buttonpanel names={['1', '2', '3', '+']} />
+      <Buttonpanel names={['0', '.', '=']} />
```
Comment (bolah2009) [inline src/App.js:16]: The app component does not need to know about all these buttons name.
I would recommend passing a single `Buttonpanel` component then add all these button names there.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Check out the optional comments before merging. :relaxed: 

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: src/App.js (hunk 2 above)
Comment (SotoArmando) [reply src/App.js:16]: ahh thanks 👍

**Pull request merged**